modules/renter/contractor/uptime.go

```diff
@@ -37,62 +35,26 @@ func (c *Contractor) IsOffline(id types.FileContractID) bool {
 // isOffline indicates whether a contract's host should be considered offline,
 // based on its scan metrics.
 func (c *Contractor) isOffline(id types.FileContractID) bool {
```

Comment:
simplified `isOffline` to only consider the most recent scan. This allows us to respond more flexibly to 75% of our hosts suddenly going offline simultaneously